requirements.txt

```diff
@@ -2,14 +2,12 @@
 # Make sure you also install the non-Python dependencies described in
 # https://github.com/google/pytype/blob/master/CONTRIBUTING.md#pytype-dependencies.
 attrs>=21.4.0
-dataclasses; python_version < '3.7'
 importlab>=0.7
-libcst
-networkx<2.8.4
+libcst>=0.4.5
+networkx>=2.6.2
```

Comment:
I'm pretty sure we need to keep the <2.8.4; last I checked, using networkx 2.8.4 caused some odd test failure.

Reply:
So please add <2.8.4 back. >=2.6.2 will not prevent version 2.8.4 from being installed.